bids/grabbids/__init__.py

```diff
@@ -1,3 +1,7 @@
-from .bids_layout import BIDSLayout
-from .bids_validator import BIDSValidator
+import warnings
+from ..layout.bids_layout import BIDSLayout
+from ..layout.bids_validator import BIDSValidator
 __all__ = ["BIDSLayout", "BIDSValidator"]
+
+warnings.simplefilter('always', DeprecationWarning)
```

Comment:
I'd rather libraries not change the `warnings` registry. This makes it harder for programs to control their outputs.